syncback/Tiltfile

```diff
@@ -0,0 +1,30 @@
+# -*- mode: Python -*-
+
+krsync_path = os.path.join(os.getcwd(), 'krsync.sh')
+
+
+def syncback(name, k8s_object, paths, src_dir, target_dir='.', namespace=''):
+    # name (str) = resource name
+    # k8s_object (str) = thing to exec on (pod/foo, deployment/bar, etc)
+    # paths (List[str]) = paths ON CONTAINER to sync. May be files or dirs. May be absolute, or relative to src_dir
+    # src_dir = dir ON CONTAINER that paths are relative to
+    # target_dir (str) = dir on local FS to copy files back to (maybe abs or relative to Tiltfile)
+
+    if not src_dir.endswith('/'):
+        fail('src_dir must be a directory and have a trailing slash')
```

Comment:
@nicks does this seem like an ok approach (as opposed to silently tacking on a trailing slash, as opposed to just passing in the value the user gave us)? doing this b/c trailing slash is semantic in rsync, and i don't expect the average tilt user to get that nuance

Reply:
having done a bunch of work with rsync in the past, this makes total sense to me.